CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 * Add `include_existing_files` parameter to `ReplicationSetupHelper`
 
+### Changed
+* Change the per part retry limit from 5 to 20 for data transfer operations. Please note that the retry system is not considered to be a part of the public interface and is subject to be adjusted
+* After the 10th retry do not wait more than 64 seconds between retry attempts
+* After the 5th retry wait a random amount of time (up to 1s) to prevent client synchronization
```

Comment:
Did you mean `After the 5th retry increase the wait time by a random amount of time (up to 1s)` ?

Either way, I don't see it, I can only pinpoint where you add the random number when hitting max